app/views/password_resets/new.html.erb

```diff
@@ -0,0 +1,14 @@
+<main class="form-main">
+  <h1>Forgot password</h1>
+  <div class="signup-login-form">
+    <div class="form-container">
+      <%= form_for(:password_reset, url: password_resets_url) do |f| %>
+       <div class="field">
+        <%= f.label :email %>
```

Comment:
Indentation issues, Please do make sure that I do not see these indentation issues again

Reply:
done